dist/components/progress/progress.js

```diff
@@ -0,0 +1,38 @@
+define([],function() {
+
+    return {
+        "maxStep": 0
+        "currentStep": 0,
+
+        "items": [
```

Comment:
This is mock data, correct? It doesn’t match the schema though (here, "items", there, "steps").